FlorenceBML/BMLor321/BMLor321.xml

```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <altIdentifier>
                             <idno>Marrassini ms. 21</idno>
                         </altIdentifier>
+                        <altIdentifier>
+                            <idno>Assemani CCCCV</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
+                            <note>According to <bibl><ptr target="bm:bm:Assemani1719Bibliotheca"/><citedRange unit="page">431</citedRange>
```

Comment:
this note belongs to the text (STU0008Lex or something) not the manuscript

Reply:
Alright! I created a STU record with that ID and removed the information to that record.